src/overlays/actors/ovl_Bg_Umajump/z_bg_umajump.h

```diff
@@ -3,13 +3,33 @@
 
 #include "global.h"
 
+#define BG_UMAJUMP_GET_OBJECT_INDEX(thisx) (((thisx)->params >> 8) & 0xFF)
+#define BG_UMAJUMP_GET_FF(thisx) ((thisx)->params &= 0xFF)
```

Comment:
```suggestion
#define BG_UMAJUMP_GET_FF(thisx) ((thisx)->params & 0xFF)
```
Like this, like we usually do.